plugins/rebalance/utils.go

```diff
@@ -0,0 +1,33 @@
+package rebalance
+
+import "time"
+
+// TimeVal represents the time value
+type TIMEVAL struct {
+	TV_sec  uint64
+	TV_usec uint64
+}
+
+var (
+	hash uint64
+)
+
+func glusterdVolinfoResetStats(r RebalanceInfo) {
+	r.RebalanceFiles = 0
+	r.RebalanceData = 0
+	r.LookedupFiles = 0
+	r.RebalanceFailures = 0
+	r.RebalanceTime = 0
+	r.SkippedFiles = 0
+}
+
+func setCommitHash(r *RebalanceInfo) {
```

Comment:
Please copy the comments from gd_set_commit_hash so we know why is being done this way.